common/thorhelper/thorsoapcall.cpp

```diff
@@ -2201,7 +2202,8 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
             }
             if (read)
                 response.append(read,payload);
-            if (payloadsize) {  // read directly into response
+            if (payloadsize) // read directly into response
+            {
                 while (read<payloadsize) {
                     checkTimeLimitExceeded(&remainingMS);
                     checkRoxieAbortMonitor(master->roxieAbortMonitor);
```

Comment:
@mckellyln does the line below need a similar change? on line 2216

Reply:
I did not think so because the loop is checking for a specific size.

Reply:
Yes, updated code.